src/Administration/Resources/app/administration/src/module/sw-settings/page/sw-settings-index/sw-settings-index.spec.js

```diff
@@ -194,63 +194,6 @@ describe('module/sw-settings/page/sw-settings-index', () => {
         );
     });
 
-    it('should render settings items in alphabetical order', async () => {
```

Comment:
@jozsefdamokos seems like you were involved in the settings page rework:
A lot of the tests here dependendet on the old structure, for now I just deleted every test that failed after removing the old code, not sure if we should bring back some tests and adjust for the new structure of if this is fine as the new settings page is way less complex (e.g. no tabs anymore)

Reply:
Added tests for the sorting back, the group label thing would be harder to test and not really useful as the implementation is pretty dynamic there 

Also removed the backgroundEnabled prop as it is useless now